.github/workflows/manual_update_development.yml

```diff
@@ -22,3 +22,25 @@ jobs:
           git fetch
           git reset --hard origin/development
           mdbook build -d docs
+
+  job_two:
+    needs: job_one
+    name: Wait for Website Update
+    runs-on: ubuntu-latest
+    steps:
+    - name: Sleep and Run
+      id: wait-deploy
+      run: |
+        echo "Sleeping for 30"
+        sleep 30
+        
+  job_three:
+    needs: [job_one, job_two]
```

Comment:
I think we should take the opportunity to use descriptive names for the jobs. Also, depending on job_two implies depending on job_one.

Reply:
OK Good. We can do:

- job_deploy
- job_wait
  - needs job_deploy
- job_checklinks
  - needs job_wait